apps/client/pages/login.tsx

```diff
@@ -1,20 +1,115 @@
-import { useAuth0 } from '@auth0/auth0-react'
-import { LoadingSpinner } from '@maybe-finance/design-system'
+import { useState, type ReactElement } from 'react'
+import { FullPageLayout } from '@maybe-finance/client/features'
+import { Input, InputPassword, Button } from '@maybe-finance/design-system'
+import { signIn, useSession } from 'next-auth/react'
 import { useRouter } from 'next/router'
 import { useEffect } from 'react'
+import Script from 'next/script'
+import Link from 'next/link'
 
 export default function LoginPage() {
-    const { isAuthenticated } = useAuth0()
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const [isValid, setIsValid] = useState(false)
+    const [errorMessage, setErrorMessage] = useState<string | null>(null)
+
+    const { data: session } = useSession()
     const router = useRouter()
 
     useEffect(() => {
-        if (isAuthenticated) router.push('/')
-    }, [isAuthenticated, router])
+        if (session) router.push('/')
```

Comment:
Can you add curly braces around the if? I'm personally not a fan of omitting them and we should create a linting / formatting rule to enforce them.